python/tests/.ipynb_checkpoints/test_preprocess-checkpoint.py

```diff
@@ -0,0 +1,126 @@
+""" Test suite for pre processing routines."""
```

Comment:
make sure this .ipynb is also gone

Reply:
Deleted

Reply:
deleted... but I really gotta find a permanent way to stop generating these checkpoints.